target_chains/ethereum/contracts/forge-test/Entropy.t.sol

```diff
@@ -87,12 +87,13 @@ contract EntropyTest is Test, EntropyTestUtils {
         bool useBlockhash
     ) public returns (uint64 sequenceNumber) {
         vm.deal(user, fee);
-        vm.prank(user);
+        vm.startPrank(user);
```

Comment:
this was a bug. i think the prior prank only applied to the first call, which is actually constructUserCommitment